src/digitalmarketplace/components/new-framework-banner/template.njk

```diff
@@ -0,0 +1,15 @@
+{%- from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner -%}
+
+{% set html %}
+  <p class="govuk-notification-banner__heading">
+    Supplier information for Digital Outcomes 6 is not available from this log in screen.
```

Comment:
> from this log in screen

Ah - just had a thought. Where are we showing this banner? Is this text valid for all places?

Reply:
The text will now be:
_Supplier information for Digital Outcomes 6 is not available from this page. Start or continue your application for the Digital Outcomes 6 framework._